libindy/src/utils/json.rs

```diff
@@ -0,0 +1,19 @@
+extern crate serde;
+extern crate serde_json;
+
+use self::serde::{Serialize, Deserialize};
+use self::serde_json::Error;
+use std::string::String;
+
+
+pub trait JsonEncodable: Serialize + Sized {
```

Comment:
This trait was eliminated some times ago as useless, do we have some new reasons to restore it?

Reply:
As far as I know, the only reason this still exists is because this PR is so old that most the changes you've made since I started working on this haven't been brought in.

Reply:
hm, we have some merge into this PR, may this deletion is missed. Lets come back to this question after final merging latest master to the PR

Reply:
I was able to remove this without any problems. All my tests are still passing and I haven't seen any new ones fail either.